packages/common/src/store/cache/reducer.ts

```diff
@@ -89,7 +89,8 @@ const forceUpdateKeys = new Set([
   'followee_reposts',
   'followee_saves',
   'associated_wallets',
-  'associated_sol_wallets'
+  'associated_sol_wallets',
+  'premium_conditions'
```

Comment:
with scheduled releases, you can freely change from usdc to special access. sometimes there was merging behavior so a track would have both conditions. this prevents that merging behavior.